dspace-rest/src/main/webapp/static/reports/restReport.js

```diff
@@ -102,6 +105,7 @@ var Report = function() {
         itemsTitle += " (" + (offset+1) + " - " + last + suff + ")";
         $("#prev,#next").attr("disabled",true);
         $("#itemdiv h3").text(itemsTitle);
+    $("#exlimit").removeClass("red");
```

Comment:
Starting here, there's several changes in this PR that were *not* in #1934.  However, I see these changes on the `dspace-6_x` branch, so I assume they likely are necessary. Just checking if we should like this PR up to *more than one* PR that was applied to `dspace-6_x`.

Reply:
Oh, wait! These changes were in #1989 (another port PR).  So, they'd likely disappear if you rebase this branch.